alerts/openshift-container-storage-operator/HighRBDCloneSnapshotCount.md

```diff
@@ -0,0 +1,68 @@
+# HighRBDCloneSnapshotCount
+
+## Meaning
+
+The number of RBD clones or snapshots (children) of a Ceph RBD image has
+exceeded 200. This metric is derived from CSI volume metadata and reflects
+excessive cloning or snapshotting activity on a single base image, typically
+driven by automation such as backup tools or CI/CD pipelines.
+
+## Impact
+
+* Performance degradation during image operations (e.g., deletion,
+  resizing, or rollback).
+* Increased recovery time after failures due to deep clone chains.
+* Potential instability in CSI provisioning and snapshot operations.
+* Higher load on Ceph monitors and OSDs when managing large numbers of child
+  images.
+
+
+## Diagnosis
+
+1. Identify the affected RBD image using the image and radosnamespace labels
+   in the alert.
+2. Access the Rook Ceph toolbox pod:
+```bash
+oc -n openshift-storage exec -it <rook-ceph-tools-pod> -- bash
+```
+3. Check the number of children:
+```bash
+rbd children --pool <pool> [--namespace <namespace>] <image-name>
+```
+4. List snapshots (if applicable):
+```bash
+rbd snap ls --pool <pool> [--namespace <namespace>] <image-name>
+```
+5. Correlate with Kubernetes resources:
+    * Find associated PVCs: oc get pv -o jsonpath='{.spec.claimRef.name}'
+      --field-selector=spec.csi.volumeAttributes.imageName=<image-name>
+    * Check VolumeSnapshots: oc get volumesnapshot -A | grep <image-name>
+6. Determine the source:
+    * Is a backup tool (e.g., Velero) creating frequent snapshots?
+    * Is a CI/CD pipeline cloning the same image repeatedly?
+
+## Mitigation
+
+1. Clean up unused PVCs that were created from snapshots:
+```bash
+oc delete pvc <pvc-name> -n <namespace>
+```
+This triggers the CSI driver to clean up the underlying RBD clone.
+
+2. Remove orphaned snapshots (only if no clones depend on them):
```

Comment:
why this is required?

Reply:
just a cleanup process? please add your reviews on it. or else we can remove this.